paparazzi.sh

```diff
@@ -144,8 +144,15 @@ case "$1" in
         ./configure --prefix=/usr
         make -j $N_CORES
         sudo make install
+        # Do some linking
+        sudo rm /usr/lib64/libfreetype.so
```

Comment:
hehehe... true! the GCC 4.9 need similar linking : S... @rmarianski will fix this when he makes RPM packages for it https://github.com/tangrams/paparazzi/issues/17. This is more like a log track to communicate back and forth with @rmarianski